include/Map/TriggerBase.hpp

```diff
@@ -23,10 +23,15 @@ struct TriggerParams {
     /* b */
 };
 
-class TriggerBase : public SysObject {
+class A : public SysObject {
+public:
+    virtual unk32 vfunc_00(void *);
+};
+
+class TriggerBase : public A {
```

Comment:
not sure if there's other usage somewhere else but you should give it a better name lol
```suggestion
class UnkStruct_TriggerBase : public SysObject {
public:
    virtual unk32 vfunc_00(void *param1);
};

class TriggerBase : public UnkStruct_TriggerBase {
```